### PR TITLE
Download and print import items license

### LINKED ISSUE #309: Connect imported items form to LaTeX template
Once #301 is merged, we have to connect it to generate LaTeX files.

### PR DESCRIPTION
**Pull request by luksan47**
Fixes #309.

### CODE REVIEW

File: resources/lang/en/document.php [hunk 1]
```diff
@@ -13,6 +13,8 @@
     'item' => 'Item',
     'kitchen' => 'kitchen utensils (eg. dishes)',
     'license' => 'Statement',
+    'missing_items' => 'You have not added any items.',
+    'missing_personal_info' => 'Your personal information are missing. Please contanct a system administrator.',
```
Comment (kdmnk) [inline resources/lang/en/document.php:17]: typo: personal information`s`,  and `contact`

File: resources/views/latex/import.blade.php [hunk 2]
```diff
@@ -38,7 +38,7 @@
 
 \begin{enumerate}
 @foreach($items as $item)
-    \item {{ $item }}
+    \item {{ $item->name }} ({{ $item->serial_number }})
```
Comment (kdmnk) [inline resources/views/latex/import.blade.php:41]: should be displayed only if the item has a serial number (currently `( )` shown in this case)

File: resources/views/document/index.blade.php [hunk 3]
```diff
@@ -33,11 +33,11 @@
...
                             </td>
                             <td>
-                                <a href="{{-- {{ route('documents.import.download') }} --}}" type="submit" class="btn waves-effect coli">@lang('document.download')</a>
+                                <a href="{{ route('documents.import.download') }}" type="submit" class="btn waves-effect coli">@lang('document.download')</a>
                             </td>
                             <td>
                                 @can('print.print')
-                                <a href="{{-- {{ route('documents.import.print') }} --}}" type="submit" class="btn waves-effect coli blue">@lang('document.print')</a>
+                                <a href="{{ route('documents.import.print') }}" type="submit" class="btn waves-effect coli blue">@lang('document.print')</a>
                                 @endcan
```
Comment (kdmnk) [inline resources/views/document/index.blade.php:41]: Enable buttons on import page as well

**Review by kdmnk — APPROVED** [review]
Left some comments

File: resources/lang/en/document.php (hunk 1 above)
Comment (luksan47) [reply resources/lang/en/document.php:17]: Information is uncountable, the correct form is without the 's'.

**Comment by kdmnk** [comment]
And is `not enough balance` error shown?

**Comment by luksan47** [comment]
>And is not enough balance error shown?

Yes

**Pull request merged**